iceoryx_hoofs/time/include/iox/duration.hpp

```diff
@@ -358,12 +354,12 @@ class Duration
 /// @return a new Duration object
 /// @attention Since negative durations are not allowed, the duration will be clamped to 0
 // AXIVION Next Construct AutosarC++19_03-A8.4.7 : Each argument is larger than two words
+// AXIVION Next Construct AutosarC++19_03-M5.17.1 : Assigning the result of a Duration multiplication with 'operator*=' to an arithmetic type is not supported
```

Comment:
Shouldn't this be restricted by a concept aka `enable_if` in C++14? A static_assert would also be ok.

Reply:
Well, it is not implemented and therefore also cannot be used. Or do you mean to create a dummy implementation with a `static_assert(always_false_v)`?